documentation/src/environment_variables.md

```diff
@@ -4,14 +4,16 @@ Besides CLI options and the configuration values in the configuration file,
 there are some environment variables that control the behaviour of a `forest`
 process.
 
-| Environment variable            | Value                            | Default | Description                                                               |
-| ------------------------------- | -------------------------------- | ------- | ------------------------------------------------------------------------- |
-| FOREST_KEYSTORE_PHRASE          | any text                         | empty   | The passphrase for the encrypted keystore                                 |
-| FOREST_CAR_LOADER_FILE_IO       | 1 or true                        | false   | Load CAR files with `RandomAccessFile` instead of `Mmap`                  |
-| FOREST_DB_DEV_MODE              | [see here](#-forest_db_dev_mode) | current | The database to use in development mode                                   |
-| FOREST_ACTOR_BUNDLE_PATH        | file path                        | empty   | Path to the local actor bundle, download from remote servers when not set |
-| FIL_PROOFS_PARAMETER_CACHE      | dir path                         | empty   | Path to folder that caches fil proof parameter files                      |
-| FOREST_PROOFS_ONLY_IPFS_GATEWAY | 1 or true                        | false   | Use only IPFS gateway for proofs parameters download                      |
+| Environment variable            | Value                            | Default                          | Description                                                               |
```

Comment:
new issue: ban `std::env::var`, and have one `ForestVars` structure

- We should have one struct where forest environment variables are described (for end users) and accessed (in code)
- Remove `documentation/src/environment_variables.md`, and have it link to the rustdoc for the above

These means we have:
- better control over environment variables
- documentation that stays up-to-date

We could also have an approach like #3370 or using https://docs.rs/figment/latest/figment/index.html for config

Cargo did something similar last year: https://github.com/rust-lang/cargo/issues/11588

Reply:
#4182

Reply:
(nonblocking, obviously)

Reply:
It would indeed be nice, this would also allow for specifying one prefix for all the env variables, e.g. `FOREST_`. 
Though instead of having to do the plumbing for this it'd be nice to have some sort of `singleton` that could be accessed anywhere in the code.